scripts/ci-entrypoint.sh

```diff
@@ -141,9 +141,16 @@ install_addons() {
 
     # Copy the kubeadm configmap to the calico-system namespace. This is a workaround needed for the calico-node-windows daemonset to be able to run in the calico-system namespace.
     "${KUBECTL}" create ns calico-system
+    for (( i=1; i<=30; i++ ))
+    do
+        if eval "${KUBECTL}" get configmap kubeadm-config --namespace=kube-system; then
```

Comment:
do we have to do eval here? does a negative lookup not produce a non-zero exit code (we should be able to just if/else based on the exit code of the command)

Reply:
Good call, changed it to use until